src/main/resources/games/strategy/debug/logging.properties

```diff
@@ -0,0 +1,7 @@
+# Global properties
+
+handlers=games.strategy.debug.ConsoleHandler
+
+# Logger-specific properties
+
+.level=INFO
```

Comment:
For game client we will want a way to update the logging level in-game, so a player can do so without mucking with a config file.

It may be overkill to go exactly that route, I do think we will want to build in an error/feature request reporting mechanism directly into the game.

I would encourage us to keep in mind that client logging is a bit different from server side logging. Error messages we show to the user are always going to be very concerning, most players will likely save and reload whenever they see one assuming the current game is potentially corrupt (this has struck me a few times, losing TripleA games is not fun). In that loop though players are not typically reporting problems.

So we have two use-cases for client logging at the moment, "high" for when we are testing and doing dev work to see everything, and "low" for normal users to suppress anything that a user should not worry about.

We do not need to worry about client file rotation, we do not need to worry even about logging to a file so long as we guarantee the output will be available on the console window. We also do not need to worry about disk space. But again, we do need to worry that anytime we show the error console to players, they will assume their game is crashing or crashed.